### ISSUE TITLE
pretrained model selection

### ISSUE THREAD

**Issue by slala2121** [google-research/simclr#136]
In the semi-supervised learning tasks, how did you end up choosing the pretrained model checkpoint? Did you take the pretrained model when its performance  e.g., contrast accuracy/ loss converged?

**Comment by chentingpc** [comment]
the models were pretrained for a fixed number of training epochs (e.g. 800) and last checkpoints were selected.

**Comment by slala2121** [comment]
Thanks.

**Comment by slala2121** [comment]
To confirm, it seems like this "fixed" # training epochs was chosen according to when the model training loss converged?

**Comment by chentingpc** [comment]
not really, training longer could actually improve further, but it becomes very costly.
